src/transforms/repair.jl

```diff
@@ -20,6 +20,75 @@ Perform repairing operation with code `K`.
 """
 struct Repair{K} <: StatelessGeometricTransform end
 
+
+# --------------
+# OPERATION (0)
+# --------------
+
+function apply(::Repair{0}, mesh)
+  points  = vertices(mesh)
+  npoints = nvertices(mesh)
+
+  # store indicators of duplicated vertices
+  dups = fill(false, npoints)
+
+  # dictionary mapping old to new indices
+  inds = Dict{Int,Int}()
+
+  count = 0
+
```

Comment:
```suggestion
  count = 0
  dups  = fill(false, npoints)
  inds  = Dict{Int,Int}()
```

Reply:
looks like you don't like comments in the code... If you read the code two years later they are very helpful.

Reply:
I think that comments are most helpful when they give more than just a description of the line of code that comes next. It is subjective of course what characterizes a good comment, but I am just cleaning up the code in the review before we can add a single paragraph of comment at the end.